mongodbatlas/fw_resource_mongodbatlas_alert_configuration_test.go

```diff
@@ -854,26 +898,6 @@ resource "mongodbatlas_alert_configuration" "test" {
     email_enabled = false
 	roles         = ["GROUP_OWNER"]
   }
-
-  matcher {
-    field_name = ""
-    operator   = null
-    value      = ""
-  }
-
-  metric_threshold_config {
-    metric_name = ""
-    operator    = null
-    threshold   = null
-    units       = null
-    mode        = ""
-  }
-
-  threshold_config {
-    operator  = null
-    threshold = null
-    units     = null
-  }
```

Comment:
I removed the empty check because a field inside the Set became required in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1421 so users cannot add a "null" element in the Set without specifying the required param.